### PR TITLE
Add articles on powershell

### PR DESCRIPTION
**Pull request by sharma-shruti**
1. Differences between Bash and Powershell
2. List of important resources for learning Powershell

### CODE REVIEW

File: articles/Bash_vs_PowerShell.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+| UNIX commands have been fairly standard in the way that they use options but be mindful, as there are variations.  | PowerShell has the advantage of a consistent syntax structure. The cmdlets are user-friendly and readable.       |
+| A pipeline connects the STDOUT (standard output) file descriptor of the first process to the STDIN (standard input) of the second. Each pipe is independent, and simply links the STDOUT and STDIN of the adjacent processes.| Using the pipeline in PowerShell enables users to create powerful one-line commands that pass output to the next command   |
+| Bash programs are mainly written in the C programming language as is the linux kernel.    | Powershell is build on .NET framework. It has ability to leverage .NET/C# functionality without having to build an entire C# app        |
+| The user interface of Bash shell is a text-based command-line interface.   | The user interface of PowerShell is a graphical command-line interface CLI.        |
```
Comment (abhi18av) [inline articles/Bash_vs_PowerShell.md]: What's a `graphical command line interface` ?

File: articles/Bash_vs_PowerShell.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+| Bash Shell is written by Brian Fox and developed by the GNU project. It was first released in 1989.  | Jeffrey Snover is the Inventor and Architect of Windows PowerShell. It was introduced in 2006.       |
+| It is mainly prepared for Linux and Unix operating system from the first day   | It can execute on any version of Windows from Window 97 to Windows 10.      |
+| Bash treats outputs as strings  | PowerShell treats output as objects.        |
+| It passes output and input as plain text, which means it is easy for the user to move information to the next program.  | PowerShell scripts share complex data, passing entire data structures between commands. PowerShell is object-oriented scripting language.|
+| The Bash shell and command language don't offer these capabilities in Windows.   | PowerShell is designed especially for the System administrators to enables them to perform a task on remote and local windows system through full access to COM and WMI. It allows administrators to manage both the workloads windows server and Linux server where production application are hosted.       |
```
Comment (abhi18av) [inline articles/Bash_vs_PowerShell.md]: Perhaps, a line or two about `COM` and `WMI` could be added

File: articles/Bash_vs_PowerShell.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+| UNIX commands have been fairly standard in the way that they use options but be mindful, as there are variations.  | PowerShell has the advantage of a consistent syntax structure. The cmdlets are user-friendly and readable.       |
+| A pipeline connects the STDOUT (standard output) file descriptor of the first process to the STDIN (standard input) of the second. Each pipe is independent, and simply links the STDOUT and STDIN of the adjacent processes.| Using the pipeline in PowerShell enables users to create powerful one-line commands that pass output to the next command   |
+| Bash programs are mainly written in the C programming language as is the linux kernel.    | Powershell is build on .NET framework. It has ability to leverage .NET/C# functionality without having to build an entire C# app        |
+| The user interface of Bash shell is a text-based command-line interface.   | The user interface of PowerShell is a graphical command-line interface CLI.        |
+| Bash typically relies on a combination of newer tools and classic Unix utilities   | PowerShell has its own set of command-line programs.        |
```
Comment (abhi18av) [inline articles/Bash_vs_PowerShell.md]: It's worth mentioning that with `Powershell core` it's possible to use the same newer and classic Unix utilities as well.

**Review by abhi18av — CHANGES_REQUESTED** [review]

File: articles/Bash_vs_PowerShell.md (hunk 1 above)
Comment (abhi18av) [inline articles/Bash_vs_PowerShell.md]: What's a `graphical command line interface` ?

Comment (sharma-shruti) [reply articles/Bash_vs_PowerShell.md]: Oh I meant graphical as well as command-line interface. In one of the videos, I saw that ISE can be used to generate PowerShell commands using the pre-defined forms. Is this correct?

File: articles/Bash_vs_PowerShell.md (hunk 2 above)
Comment (sharma-shruti) [reply articles/Bash_vs_PowerShell.md]: Okay!

File: articles/Bash_vs_PowerShell.md (hunk 3 above)
Comment (sharma-shruti) [reply articles/Bash_vs_PowerShell.md]: I will update this information.

File: articles/Bash_vs_PowerShell.md (hunk 1 above)
Comment (abhi18av) [reply articles/Bash_vs_PowerShell.md]: Yup, the same could be achieved in Visual Studio Code, this is what's called a snippet.

**Review by abhi18av — APPROVED** [review]

**Comment by abhi18av** [comment]
@sharma-shruti , I can see that you've started to use the formatted git commit messages so here's a good styleguide on how it's done at scale and across teams.

http://udacity.github.io/git-styleguide/

CC @arushi-bhatt

**Pull request merged**